src/Elastic.Apm.AspNetCore/AspNetCoreLogger.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Elastic.Apm.Logging;
+using Microsoft.Extensions.Logging;
+using LogLevel = Elastic.Apm.Logging.LogLevel;
+
+namespace Elastic.Apm.AspNetCore
+{
+	public class AspNetCoreLogger : IApmLogger
+	{
+		public AspNetCoreLogger(ILoggerFactory loggerFactory)
+		{
+			_logger = loggerFactory?.CreateLogger("apm") ?? throw new ArgumentNullException(nameof(loggerFactory));
+		}
+
+		private readonly ILogger _logger;
+
+		public LogLevel Level { get; } = LogLevel.Error;

```

Comment:
Agreed with you. Error was a simple solution for draft pull request. Also, I agreed with the 2nd option (use log level from standard asp.net core config section). However, fetch logger configuration isn't a simple task in asp.net core due to it can be configured in various way. By this reson, I iterate over `IsEnabled` method to find out min level which is enabled for logger.

Reply:
> However, fetch logger configuration isn't a simple task 

Yes, I also tried to come up with something, but that is kind of not part of the API. One thing comes to my mind which may be a problem: Is the loglevel changeable on the fly while the application is running? Atm the code caches the value in the `AspNetCoreLogger` construction, and I wonder if this can change. 
On the other hand reading it always, when the `LogLevel` property is accessed sounds dangerous in terms of overhead (if we go there, we can just measure this).

But overall I think we are good to processed here, this should not block you, we may rethink this part, but it's ok for now.